### PR TITLE
Fix the heading number for the next show and tell

### PR DESCRIPTION
**Pull request by tuzz**
I wrote a quick thing to do a more exhaustive check:

```ruby
# If a show and tell writeup references the next meeting, check
# that its number is one greater than the number in the filename.

Dir.glob("soups/show-and-tell/*").each do |path|
  File.readlines(path).each do |line|
    next unless line.start_with?("#")
    next unless line.downcase.include?("show")
    next unless line.downcase.include?("tell")
    next unless line.match(/\d+/)

    expected = Integer(path[/\d+/]) + 1
    actual = Integer(line[/\d+/])

    unless expected == actual
      puts "#{path} might have the wrong heading #{line}"
    end
  end
end
```

It detects one other discrepancy in `show-and-tell-50` but that intentionally references `54` because it was written up several meetings later.

**TL;DR** everything looks good. 👍

### CODE REVIEW

**Comment by floehopper** [comment]
Thanks, Chris! 👍
